build-system/babel-config/pre-closure-config.js

```diff
@@ -34,7 +34,6 @@ function getPreClosureConfig() {
     {
       imports: {
         // Imports that are not needed for valid transformed documents.
-        '../build/ampshared.css': ['cssText', 'ampSharedCss'],
```

Comment:
I thought we were going to steal the styles from the amp document? `document.querySelector('style[amp-runtime]').textContent`

Reply:
yep, just wanted to test this out. will push out the change here plus tests

Reply:
I too am confused, we don't intend to merge this right?

Reply:
ive pushed out the latest code. sorry for the wait, been trying to figure out the tests for these.